src/solc_0.8/defi/ERC20RewardPool.sol

```diff
@@ -128,7 +128,12 @@ contract ERC20RewardPool is
     /// @dev this function must be called in an emergency situation only.
     /// @dev Calling it is risky specially when rewardToken == stakeToken
     function recoverFunds(address receiver) external onlyRole(DEFAULT_ADMIN_ROLE) isValidAddress(receiver) {
-        rewardToken.safeTransfer(receiver, rewardToken.balanceOf(address(this)));
+        uint256 recoverAmount = rewardToken.balanceOf(address(this));
```

Comment:
put this line in the else of the condition

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/664/commits/6aa403dd05c0674c05b8a3f8c15124925b848451